IND/MarketIndices/EquityIndexExampleIndividuals.rdf

```diff
@@ -104,12 +105,49 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/Indicators/Indicators/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/InterestRates/InterestRates/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/MarketIndices/BasketIndices/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/EquitiesExampleIndividuals/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/Baskets/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/SecuritiesClassification/"/>
 		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/MarketIndices/EquityIndexExampleIndividuals/"/>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Provisional"/>
 	</owl:Ontology>
 	
+	<owl:NamedIndividual rdf:about="&fibo-ind-mkt-eqind;DowJonesIndustrialAverage">
+		<rdf:type rdf:resource="&fibo-ind-mkt-bas;EquityIndex"/>
+		<rdfs:label>Dow Jones Industrial Average</rdfs:label>
+		<skos:definition>equity index of 30 substantial stocks that are traded on the New York Stock Exchange (NYSE) and the Nasdaq</skos:definition>
+		<fibo-be-fct-pub:hasPublisher rdf:resource="&fibo-ind-mkt-eqind;SPDowJonesIndices"/>
+		<fibo-fbc-dae-dbt:isBasedOn rdf:resource="&fibo-ind-mkt-eqind;DowJonesIndustrialAverageBasket"/>
+		<fibo-fnd-utl-alx:isCalculatedViaMethodology xml:lang="en">The Averages Committee, which includes the managing editor of The Wall Street Journal, the head of Dow Jones Indexes research, and the head of CME Group research, determine which components become a part of the DJIA. There are no set rules the Averages Committee must follow when selecting Dow components, only broad parameters to ensure those companies represent a large portion of the overall economic performance in the United States. Each average is reviewed at least once annually, but composition changes are rare for the sake of continuity. Since its beginnings in May of 1896, the Dow Jones stocks list has only changed 54 times. As of June 2018, the last remaining company of the original 12 components of the DJIA, General Electric (GE), was replaced by Walgreens Boots Alliance.</fibo-fnd-utl-alx:isCalculatedViaMethodology>
```

Comment:
I'll add a note under IND-77.  The content for this pull request was originally developed under IND-77, which is incomplete.  We agreed that it was incomplete, and that I should do a pull request for SEC-74 to cover all of the individuals, but it would have been difficult to remove these updates since they were done out of sequence.

Reply:
I'd be happier if the property was empty - currently it's not using the property as intended and so misleading: the text that's there should be on the basket.

Reply:
OK with handling this in the IND-77 change that is forthcoming.